src/main/java/frc/robot/Robot.java

```diff
@@ -67,6 +69,7 @@ public void robotInit() {
     public void robotPeriodic() {
         CommandScheduler.getInstance().run();
         dashboardDataContainer.update();
```

Comment:
dashboardDataContainer.update();
In this method comment out or delete: `dashboardController.update();`
and make DashboardController `@deprecated`
We dont need it anymore with Oblog.